backend/src/polling_station/structs.rs

```diff
@@ -370,10 +370,16 @@ pub struct PollingStationStatusEntry {
     pub status: PollingStationStatus,
 }
 
-#[derive(Debug, Serialize, Deserialize, ToSchema, sqlx::Type, Eq, PartialEq)]
+#[derive(Debug, Serialize, Deserialize, ToSchema, sqlx::Type, Eq, PartialEq, Clone)]
+#[serde(rename_all = "snake_case")]
 pub enum PollingStationStatus {
-    Incomplete,
-    Complete,
+    Correction,
+    Definitive,
+    Difference,
+    ExtraEntry,
+    FirstEntry,
+    Objections,
+    SecondEntry,
```

Comment:
I'd prefer limiting the statuses to the ones that are relavant to / reachable by what we've built so far.

Reply:
Why? They were already present in the frontend as labels for `<Badge>`. Now we have it concisely typed throughout

Reply:
Agree that it's better to have it concisely typed throughout, but 5 out of those 7 statuses are dead code. And our API specs are now incorrect, since they list those 5 statuses.

This (including that `<Badge>` existed at all) ties back into the bigger and older discussion of how we built UI placeholders. That's not a discussion to drag into this PR, though.

My prefered solution would be to clean up the statuses throughout the whole stack, reducing them to the ones supported by the functionality, so `FirstEntry` and `Definitive`.